lms/djangoapps/commerce/utils.py

```diff
@@ -32,3 +37,19 @@ def audit_log(name, **kwargs):
     message = u'{name}: {payload}'.format(name=name, payload=payload)
 
     log.info(message)
+
+
+class EcommerceService(object):
+    """ Helper class for ecommerce service integration. """
+    def __init__(self):
+        self.config = CommerceConfiguration.current()
+
+    def is_enabled(self):
+        """ Check if the service is enabled and that it's not a microsite. """
+        return self.config.checkout_on_ecommerce_service and not microsite.is_request_in_microsite()
```

Comment:
Is that a functionality that will be used in the future for microsites? Because it's practically the same thing: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/theming/helpers.py#L34